hack/ci/operator-olm-test.sh

```diff
@@ -33,8 +33,8 @@ fi
 csv_file=${OUTPUT}/nexus-operator-m88i/${OP_VERSION}/nexus-operator.v${OP_VERSION}.clusterserviceversion.yaml
 echo "---> Updating CSV file '${csv_file}' to imagePullPolicy: Never"
 sed -i 's/imagePullPolicy: Always/imagePullPolicy: Never/g' ${csv_file}
-echo "---> Resulting imagePullPolicy"
-cat ${csv_file} | grep -irn imagePullPolicy
+echo "---> Resulting imagePullPolicy on manifest files"
+grep -rn imagePullPolicy ${OUTPUT}/nexus-operator-m88i
```

Comment:
I forgot to comment this one on my previous review, it makes sense to me (reducing.the.amount.of.processes++)